00-supplement-database-design.md

```diff
@@ -30,5 +30,10 @@ Other database management systems
     * Multiple simultaneous users
 	* More difficult to setup and maintain
 
+* Microsoft SQL Server (SQL Server)
+    * Multiple simultaneous users
+	* Easy to use
+	* Most complex part is development and troubleshooting
+
```

Comment:
Depending on where and/or the database is hosted, it can allow for multiple users to access the database through different platforms (for example, I could access the database from a GUI created in C# and through management studio. There aren't strict locks preventing such transactions from being executed across different platforms)

Reply:
Depending on where and/or if the database is hosted, it can allow for multiple users to access the database through different platforms (for example, I could access the database from a GUI created in C# and through management studio. There aren't strict locks preventing such transactions from being executed across different platforms)